### PR TITLE
Variant-Alt tweaks

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: style_editor.html [hunk 1]
```diff
@@ -99,15 +99,19 @@
...
-  padding: 2px;
-  display: inline-block;
+  border-radius: 4px;
+  font-size: 18px;
+  display: inline-flex;
   color: black;
   white-space: nowrap;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  transform: scale(.8);
```
Comment (profezzorn) [inline style_editor.html:112]: Whut?

File: style_editor.html [hunk 2]
```diff
@@ -99,15 +99,19 @@
...
+  margin-left: -35px;
```
Comment (profezzorn) [inline style_editor.html:113]: Wat?

**Review by profezzorn — COMMENTED** [review]
I think you might have to explain what the CSS is doing here.
Perferably in commments....

**Comment by NoSloppy** [comment]
> I think you might have to explain what the CSS is doing here. Preferably in comments....

Fixed. Turns out I had Chrome "workspaces" enabled and it was writing to my local file as I changed and played with CSS in the Dev Tools inspector!
Not great.

**Pull request merged**